src/main/java/org/broadinstitute/consent/http/service/SupportRequestService.java

```diff
@@ -10,27 +10,18 @@
 import com.google.inject.Inject;
 import jakarta.ws.rs.ServerErrorException;
 import java.nio.charset.StandardCharsets;
-import java.util.Objects;
 import org.broadinstitute.consent.http.configurations.ServicesConfiguration;
-import org.broadinstitute.consent.http.db.InstitutionDAO;
-import org.broadinstitute.consent.http.db.UserDAO;
-import org.broadinstitute.consent.http.models.User;
-import org.broadinstitute.consent.http.models.UserUpdateFields;
 import org.broadinstitute.consent.http.models.support.SupportTicket;
-import org.broadinstitute.consent.http.models.support.SupportTicketCreator;
 import org.broadinstitute.consent.http.util.ConsentLogger;
 import org.broadinstitute.consent.http.util.HttpClientUtil;
 
 public class SupportRequestService implements ConsentLogger {
```

Comment:
With this change, there are no uses left of `SupportRequestService` outside of tests. Is it being kept around because it will be used in an upcoming PR?

It's not clear to me why `handleInstitutionSOSupportRequest` was removed but `postTicketToSupport` was not, as both are unused on your branch.

Reply:
> With this change, there are no uses left of `SupportRequestService` outside of tests. Is it being kept around because it will be used in an upcoming PR?

Yes, exactly

> It's not clear to me why `handleInstitutionSOSupportRequest` was removed but `postTicketToSupport` was not, as both are unused on your branch.

`postTicketToSupport` is functionality we need for the ticket.